pkg/globalnet/controllers/global_ingressip_controller.go

```diff
@@ -199,3 +199,14 @@ func (c *globalIngressIPController) onDelete(ingressIP *submarinerv1.GlobalIngre
 		return nil
 	}, ingressIP.Status.AllocatedIP)
 }
+
+func GetIngressIPName(obj runtime.Object) string {
```

Comment:
```suggestion
func getIngressIPName(obj runtime.Object) string {
```

Reply:
We'll need to call it from `_test.go` code too.